.github/workflows/action-linter.yaml

```diff
@@ -0,0 +1,72 @@
+name: GitHub Actions Linter
+on: pull_request
+jobs:
+  lint-a-action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - id: setup-actionlint
+        name: setup actionlint
+        uses: ./.github/actions/actionlint
+        
+      # SC2016 incorect because of the markdown format & SC2086 incorect when setting git a env variable 
```

Comment:
I don't understand the meaning of the SC2086 comment

Reply:
I've updated the message. 

The SC2016 rule try to avoid that we use simple quote to work with shell variable but in our case we don't want the var to be expended because they are part of the `-format` of the action linter.
It's a false positive 

I didn't find any other way to ignore this one line